doc/release_notes/release_notes_1.2.rst

```diff
@@ -0,0 +1,394 @@
+- :acrn-issue:`2934` - Use virtual APIC IDs for Pre-launched VMs
+- :acrn-issue:`2947` - Debug: GuestOS Launch inhibit
+- :acrn-issue:`2965` - Remove need for init_fallback_iommu_domain and fallback_iommu_domain
+- :acrn-issue:`2987` - HV: vuart refine
+- :acrn-issue:`3012` - improve acrn-dm logging system, to make it easy to debug.
+- :acrn-issue:`3039` - Add scenario for industry use case
+- :acrn-issue:`3051` - Kata Container Storage: Support rescan feature for virtio-blk
+- :acrn-issue:`3061` - Rename some variables to be more readable
+- :acrn-issue:`3107` - isolate acpi parse code
+- :acrn-issue:`3130` - Clean up spinlock
+- :acrn-issue:`3163` - WaaG: USB host support
+- :acrn-issue:`3189` - DM: handle virtio-console writes when no socket backend is connected
+- :acrn-issue:`3227` - Cleanup vIOAPIC and vPIC for RT VM
+- :acrn-issue:`3232` - DM: support VMs communication with virtio-console
+- :acrn-issue:`3253` - Introduce vLAPIC state for VMs
+- :acrn-issue:`3263` - WBINVD virtualization for realtime VM impact
+- :acrn-issue:`3317` - New Intel CPU Flaw MDS Microarchitectural Data Sampling
+- :acrn-issue:`3357` - Support virtio-i2c
+- :acrn-issue:`3401` - When launch or reboot UOS execute "fdisk -l" to query disk information with multiple USB devices,It will be caton phenomenon
+- :acrn-issue:`3402` - Prepare uuids for kata
+- :acrn-issue:`3420` - change param type of init_pcpu_pre
+- :acrn-issue:`3446` - rename board name of nuc7i7bnh to nuc7i7dnb
+- :acrn-issue:`3459` - dm: support VMs communication with virtio-console
+- :acrn-issue:`3484` - tools: support force stop VM
+- :acrn-issue:`3486` - USB mediator: mediator mode can not recongnize the SSD disk in RTVM and LaaG
+- :acrn-issue:`3497` - Inject exception for invalid vmcall
+- :acrn-issue:`3498` - Return extended info in vCPUID leaf 0x40000001
+- :acrn-issue:`3501` - Virtualization supports Always Running Timer (ART)
+- :acrn-issue:`3505` - Succeed to reboot the whole system while execute reboot command in SOS
+- :acrn-issue:`3506` - OVMF release v1.2
+
+Known Issues
+************
+
+:acrn-issue:`3465` - HV: reinit pbar base if a device reset is detected
+   When a passthru pci device is reset, its physical bar base address may be reset to 0,
+   with vpci bar emulation, vpci needs to reinit the physical bar base address to a
+   valid address if a device reset is detected.
+
+   **Impact:** Fail to launch Clear Linux Preempt_RT VM with ``reset`` passthru parameter
+
+   **Workaround:** Issue resolved on ACRN tag: ``acrn-2019w33.1-140000p``
+
+-----
+
+:acrn-issue:`3520` - bundle of "VGPU unconformance guest" messages observed for "gvt" in SOS console while using UOS
+   After the need_force_wake is not removed in course of submitting VGPU workload,
+   it will print a bundle of below messages while the User VM is started.
+
+   | gvt: vgpu1 unconformance guest detected
+   | gvt: vgpu1 unconformance mmio 0x2098:0xffffffff,0x0
+
+   **Impact:** Mess output from the monitor
```

Comment:
Do you mean "Messy output on the monitor"?

Reply:
Yes, I have changed it to "Messy and repetitive output from the monitor"